src/main/java/dk/cngroup/lentils/service/StatusService.java

```diff
@@ -21,8 +21,10 @@ public StatusService(final StatusRepository statusRepository) {
     }
 
     public void markCypher(final Cypher cypher, final Team team, final CypherStatus cypherStatus) {
-        Status status = getStatusByTeamAndCypher(team, cypher);
-        saveNewStatus(status, cypherStatus);
+        if (cypher != null) {
```

Comment:
Co se tady osetruje za problem? Pokud nam nekdo posle null, tak musime oretrit to dane misto a vratit patricnou vyjimku, ne az tady to zkontrolovat a kdyz mame null,tak to preskocit a ignorovat.